lib/smart_answer/erb_renderer/format_capture_helper.rb

```diff
@@ -13,7 +13,7 @@ def text_for(name, &block)
       content = capture_content(&block)
       content = strip_leading_spaces(content)
       content = normalize_blank_lines(content)
-      content_for(name, content.strip)
+      content_for(name, content.strip.html_safe)
```

Comment:
Using `html_safe` makes me a little nervous, but I can't see any place in the code where user text input would passed through this. It would be nice to use [`sanatize`](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-sanitize) to do a more targeted "fix", but there doesn't seem to be a straight forward way to do that when it's character replacement rather than a tag or attribute being sanitized. So this does look like the best option.